docs/installation.rst

```diff
@@ -62,6 +62,47 @@ Install conda via anaconda or miniconda (no root privileges required)
         conda -c conda-forge boost
         conda -c anaconda cmake
 
+**Installing dependencies via spack**
+
+``spack`` is a package manager which was designed for HPC software. It is language agnostic and
+compiles binaries from source. See quote from ``spack``'s `documentation
+<https://spack.readthedocs.io/en/latest/>`_ below:
+
+   Spack is a package management tool designed to support multiple versions and configurations of
+   software on a wide variety of platforms and environments. It was designed for large supercomputing
+   centers, where many users and application teams share common installations of software on clusters
+   with exotic architectures, using libraries that do not have a standard ABI. Spack is
+   non-destructive: installing a new version does not break existing installations, so many
+   configurations can coexist on the same system.
+
+To install ``spack`` please follow instruction from their documentation (see `install instructions
+<https://spack.readthedocs.io/en/latest/getting_started.html#installation>`_).
+
+In the root directory of the repository there is a ``spack.yaml`` file which can be used to install
+all required dependencies. The following command will create a ``spack`` environment (called
+``nextsim`` in this example) and install the dependencies automatically:
+
+.. code::
+
+   spack env create nextsim spack.yaml
+
+.. note::
+
+   This will install by building from source. This can take a significant amount of time, depending
```

Comment:
Running the `env create` command does not start the install, I think. You need to activate the environment, and then run `spack install` for the install to actually start.

Reply:
Thanks for spotting this. I have updated the instructions :ok_hand: